text/0000-optimize-by-default.md

```diff
@@ -0,0 +1,166 @@
+- Start Date: 2015-03-10
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+Make rustc and cargo produce optimized binaries by default.
+
+# Motivation
+
+
+Optimizing by default is a safety & usability feature that everyone benefits
```

Comment:
I feel like you're using "safety" in a manner very different to how it is usually used for Rust, since these defaults are surely unrelated to memory safety (or any sort of "code" safety in general)?